README.md

```diff
@@ -1,8 +1,83 @@
-# React + Vite
+# FE NC News
 
-This template provides a minimal setup to get React working in Vite with HMR and some ESLint rules.
+This is a news application built with React for the frontend, using React Router for navigation and Context API for state management. Axios is used for API calls. The platform allows users to read and interact with articles on various topics.
 
-Currently, two official plugins are available:
+## Deployed Version
 
-- [@vitejs/plugin-react](https://github.com/vitejs/vite-plugin-react/blob/main/packages/plugin-react/README.md) uses [Babel](https://babeljs.io/) for Fast Refresh
-- [@vitejs/plugin-react-swc](https://github.com/vitejs/vite-plugin-react-swc) uses [SWC](https://swc.rs/) for Fast Refresh
+You can access the deployed version of the app here: [FE NC News](https://https://fe-nc-news-gui8.onrender.com/)
```

Comment:
^^ you need to remove the first https:// :)

Reply:
Thank you for pointing this out Niamh! Will fix it now. :blush: :raised_hands: